README.md

```diff
@@ -125,6 +125,58 @@ The needed GitHub app permissions are the following:
+| `DEPENDABOT_CONFIG_FILE`   | False                                           | ""                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Location of the configuration file for `dependabot.yml` configurations. If the file is present locally it takes precedence over the one in the repository.                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+
+### Private repositories configuration
+
+Dependabot allows the configuration of [private registries](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#configuration-options-for-private-registries) for dependabot to use.  
+To add a private registry configuration to the dependabot file the `DEPENDABOT_CONFIG_FILE` needs to be set with the path of the configuration file.
+
+This configuration file needs to exist on the repository where the action runs. It can also be created locally to test some configurations (if created locally it takes precedence over the file on the repository).
+
+#### Usage
+
+Set the input variable:
+
+```yaml
+DEPENDABOT_CONFIG_FILE = "dependabot-config.yaml"
+```
+
+Create a file on your repository in the same path:
+
+```yaml
+npm:
+  type: "npm"
+  url: "https://yourprivateregistry/npm/"
+  username: "${{secrets.username}}"
+  password: "${{secrets.password}}"
+  key: <used if necessary>
+  token: <used if necessary>
+  replaces-base: <used if necessary>
+maven:
+  type: "maven"
+  url: "https://yourprivateregistry/maven/"
+  username: "${{secrets.username}}"
+  password: "${{secrets.password}}"
+```
+
+The principal key of each configuration need to match the package managers that the [script is looking for](https://github.com/github/evergreen/blob/main/dependabot_file.py#L78).
```

Comment:
Can we change this link to a specific SHA(permalink)? If we add any changes to that file line 78 won't be the same.

Reply:
I agree @jmeridth , for example now it's already moved to another place. Should we add this page instead from the oficial documentation?
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem
Even though currently the script does not support all the available packages I believe the missing ones shouldn't be an issue to add.
Checking the list the supported `package-ecosystems` still not added to the script are:
- devcontainers
- elm
- gitsubmodule
- gradle
- pub
- swift

Following the same logic that is now implemented it shouldn't be too much dificult to add new supported packages.
Thanks